hydra-cluster/src/Hydra/Cluster/Scenarios.hs

```diff
@@ -339,6 +342,61 @@ canCloseWithLongContestationPeriod tracer workDir node@RunningNode{networkId} hy
     (fuelUTxO, otherUTxO) <- queryMarkedUTxO node actorVk
     traceWith tracer RemainingFunds{actor = actorName actor, fuelUTxO, otherUTxO}
 
+canSubmitUserTransaction ::
+  Tracer IO EndToEndLog ->
+  FilePath ->
+  RunningNode ->
+  TxId ->
+  IO ()
+canSubmitUserTransaction tracer workDir node hydraScriptsTxId =
+  (`finally` returnFundsToFaucet tracer node Alice) $ do
+    refuelIfNeeded tracer node Alice 25_000_000
+    aliceChainConfig <- chainConfigFor Alice workDir nodeSocket [] $ UnsafeContestationPeriod 100
+    let hydraNodeId = 1
+    withHydraNode tracer aliceChainConfig workDir hydraNodeId aliceSk [] [1] hydraScriptsTxId $ \_ -> do
+      -- let's prepare a _user_ transaction from Bob to Carol
```

Comment:
i think this comment is wrong or not needed

Reply:
what I wanted to say is that we are preparing a tx to send something from Bob to Carol and bob here is a hydra-node _user_ using this new feature if that makes sense :) .